src/App.jsx

```diff
@@ -1,33 +1,56 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function getPreparedMovies(movies, { query }) {
+  let preparedMovies = [...movies];
+
+  if (query) {
+    preparedMovies = preparedMovies
+      .filter(movie => movie.title.toLowerCase()
+        .includes(query.toLowerCase().trim())
+        || movie.description.toLowerCase()
+          .includes(query.toLowerCase().trim()));
+  }
+
+  return preparedMovies;
```

Comment:
```suggestion

  if (!query) {
    return movies
  }
  
    return preparedMovies
      .filter(movie => movie.title.toLowerCase()
        .includes(query.toLowerCase().trim())
        || movie.description.toLowerCase()
          .includes(query.toLowerCase().trim()));
```
no need to create separate variable.
method filter returns filtered array.